problem_builder/step.py

```diff
@@ -81,6 +83,13 @@ class MentoringStepBlock(
         scope=Scope.content
     )
 
+    # User state
+    student_results = List(
```

Comment:
@itsjeyd this is unlikely to work at all - how do you than distinguish the questions when you need to display their values? By sequential number? I'm not sure it would work (but not sure it wouldn't as well). Let's keep it as is for now, but please keep in mind it might make sense to use Dict.